cves/kernel/CVE-2021-3178.yml

```diff
@@ -167,8 +170,11 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: |
+    An automated tool could likely have found this vulnerability, as it is
+    a network attack vector and a low complexity attack. The only caveat is
+    that high (kernel) permissions are required.
```

Comment:
> The only caveat is that high (kernel) permissions are required.

Where did you find this? It makes logical sense that not just any user can exploit this vulnerability but I was wondering if this was explicitly written anywhere or if you had something that made you believe this

Reply:
https://cve.report/CVE-2021-3178 
https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?name=CVE-2021-3178&vector=AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:N&version=3.1&source=NIST

The CVE shows permissions required as high. I also believe I read kernel permissions were required but I do not remember where.